nio_cli/commands/config.py

```diff
@@ -26,8 +28,61 @@ def config_project(name='.'):
                 tmp.write('PK_TOKEN={}\n'.format(pk_token))
             else:
                 tmp.write(line)
-        os.remove(env_location)
-        os.rename(tmp.name, env_location)
+        os.remove(conf_location)
+        os.rename(tmp.name, conf_location)
+    secure = input('Optional secure instance configuration [Y/N]: ')
+    if secure.lower() == 'y':
+        config_ssl(name, conf_location)
+
+
+def config_ssl(name, conf_location):
+
+    ssl_cert = ''
+    ssl_key = ''
+    new_certs = input('Generate a self-signed certificate/key [Y/N]: ')
+
+    if (new_certs.lower() == 'y'):
+        v = ssl.OPENSSL_VERSION.split()[0]
+        if v != 'OpenSSL':
+            print('No OpenSSL installation detected. Your instance has still been configured but no certs were installed. To install certificates install OpenSSL and re-run "nio configure"')
+            return
+
+        subprocess.call('mkdir ssl &&  cd ssl', shell=True)
```

Comment:
I don't think this is actually changing the directory to ssl. At least in my test it creates an `ssl` directory and there's nothing in it. The key gets created in the directory I ran the CLI from.

I think the ideal behavior is that the key gets created in the project directory anyway, we don't need to be creating folders on the user's machine